modules/cert-manager-remove-resources-console.adoc

```diff
@@ -27,14 +27,16 @@ Optionally, after uninstalling the {cert-manager-operator}, you can remove its r
 
 *** `Certificate`
 *** `CertificateRequest`
-*** `CertManager` (`config.openshift.io`)
+// *** `CertManager` (`config.openshift.io`)
```

Comment:
@thejasn @swghosh Is it correct that there is only one CertManager resource now (and that it's the operator.openshift.io one?)